examples/package.json

```diff
@@ -1,13 +1,15 @@
 {
-  "name": "apollo-voyager-example",
+  "name": "voyager-server-example",
   "version": "0.2.7-rc3",
   "private": true,
   "description": "",
   "scripts": {
     "compile": "exit 0",
     "watch": "exit 0",
     "compile:clean": "exit 0",
-    "test-examples": "ava --concurrency=1 test/*.test.js"
+    "test-examples": "ava --concurrency=1 test/*.test.js",
+    "push-basic": "./push-script.sh $npm_package_version basic",
+    "push-keycloak": "./push-script.sh $npm_package_version keycloak"
```

Comment:
sorry for my lack of knowledge, is `npm_package_version` set by npm automatically?

Reply:
yes it is.